### DISCUSSION TITLE
JVM error

### DISCUSSION THREAD

**Discussion by gian9959** [itzg/docker-minecraft-server#1782] [Q&A] [answered]
I'm trying to set up a small server, but when I set it up using docker compose the JVM gives me this error and I have no idea how to fix it:
![error](https://user-images.githubusercontent.com/95230498/195658878-a9853ecc-4664-48a5-bfab-fcd733c5b9a7.png)

This is the compose file:
![dockerCompose](https://user-images.githubusercontent.com/95230498/195659912-4f9c0c24-6c44-4555-a799-f5d41bdbd1d3.png)

This is the environment file:
![env](https://user-images.githubusercontent.com/95230498/195659945-d5ef0733-2325-46d2-a92a-cac8e9f0e501.png)

**Comment by itzg** [comment, marked as answer]
You'll need to use the image `itzg/minecraft-server:java8-multiarch` for that version of Minecraft.

https://github.com/itzg/docker-minecraft-server#running-minecraft-server-on-different-java-version

**Reply by gian9959** [reply]
thank you so much, I am so dumb I didn't catch this
